backend/src/api/model/event.rs

```diff
@@ -102,6 +102,22 @@ impl Event {
 }
 
 impl Event {
+    pub(crate) async fn load(context: &Context) -> ApiResult<Vec<Self>> {
```

Comment:
Would probably call it differently. `load_all` or sth? `Event::load(...)` doesn't convey that all events are loaded IMO

Reply:
It's `Series::load` as well, though. Should we change that as well?

Reply:
I'd say yes.